src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -143,4 +143,175 @@
   <data name="InvalidOperation_RemoveFail" xml:space="preserve">
     <value>Remove access rule failed</value>
   </data>
-</root>
+  <data name="ArgumentNull_Buffer" xml:space="preserve">
```

Comment:
These are all resource strings consumed by the newly added files coming from IO.FileSystem.